config/routes.rb

```diff
@@ -18,6 +18,10 @@
   get '/login', to: 'welcome#login'
 
   get '/building_map', to: 'building_map#index'
+  get '/building_map_route', to: 'building_map#route'
+  get '/building_map_markers', to: 'building_map#markers'
+  get '/building_map_buildings', to: 'building_map#buildings'
+  get '/building_map_view', to: 'building_map#view'
```

Comment:
Just a suggestion. Maybe we could make those links look a little more beautiful?
```suggestion
  get '/building_map/route', to: 'building_map#route'
  get '/building_map/markers', to: 'building_map#markers'
  get '/building_map/buildings', to: 'building_map#buildings'
  get '/building_map/view', to: 'building_map#view'
```
On the other hand, of course nobody should ever navigate to them directly via url, so you can just leave it as it is if you want.

Reply:
Done!